### PR TITLE
Store model name in elasticsearch for filterable lists

### PR DESCRIPTION
**Pull request by alexm118**
It turns out, on our enforcement and event pages, we limit the result set to specific class type, this allows us to implement the same logic with elasticsearch.

---

## Additions

- model_class field to elasticsearch index
- Additional filter on ES results to limit them to a specific class when applicable

## Removals

-

## Changes

-

## How to test this PR

1. Search enforcement actions and ensure only EnforcementActionPage results return.

## Notes and todos

-

## Checklist

- [ ] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Future todos are captured in comments and/or tickets
- [ ] Project documentation has been updated, potentially one or more of:
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### CODE REVIEW

File: cfgov/v1/forms.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def get_page_set(self):
         else:
             query = self.generate_query()
             return self.filterable_pages.filter(query).distinct().order_by(
-                '-date_published'
+                '-initial_filing_date'
```
Comment (alexm118) [inline cfgov/v1/forms.py:324]: This was a copy paste error that got introduced and needs to be cleaned up to preserve existing EnforcementAction filterable list search.

**Review by willbarton — APPROVED** [review]
Looks straight-forward to me!

**Review by higs4281 — APPROVED** [review]
LGTM

**Review by cwdavies — APPROVED** [review]
These changes look good to me

**Pull request merged**